py/desisim/test/test_quickbrick.py

```diff
@@ -0,0 +1,127 @@
+        brick = desispec.io.Brick(brickfile)
+        self.assertEqual(brick.get_num_spectra(), nspec)
+        brick.close()
+
+    #- Test quickbrick with a bunch of options
+    @unittest.skipIf('TRAVIS_JOB_ID' in os.environ, 'Skipping memory hungry quickbrick/specsim test on Travis')
+    def test_quickbrick_options(self):
+        brickname = 'test2'
+        nspec = 3
+        cmd = "quickbrick -b {} --objtype BGS -n {} --outdir {}".format(brickname, nspec, self.testdir)
+        cmd = cmd + " --airmass 1.5 --verbose --zrange-bgs 0.1 0.2"
+        cmd = cmd + " --moon-phase 0.1 --moon-angle 30 --moon-zenith 20"
+        args = quickbrick.parse(cmd.split()[1:])
+        results = quickbrick.main(args)
+        brickfile = '{}/brick-b-{}.fits'.format(self.testdir, brickname)
+        truth = fits.getdata(brickfile, '_TRUTH')        
+        z = truth['TRUEZ']
+        self.assertTrue(np.all((0.1 <= z) & (z <= 0.2)))
+
+    #- Ensure that using --seed results in reproducible spectra
+    @unittest.skipIf('TRAVIS_JOB_ID' in os.environ, 'Skipping memory hungry quickbrick/specsim test on Travis')
+    def test_quickbrick_seed(self):
+        nspec = 2
+        cmd = "quickbrick --seed 1 -b test1a --objtype BRIGHT_MIX -n {} --outdir {}".format(nspec, self.testdir)
+        quickbrick.main(quickbrick.parse(cmd.split()[1:]))
+        cmd = "quickbrick --seed 1 -b test1b --objtype BRIGHT_MIX -n {} --outdir {}".format(nspec, self.testdir)
+        quickbrick.main(quickbrick.parse(cmd.split()[1:]))
+        cmd = "quickbrick --seed 2 -b test2 --objtype BRIGHT_MIX -n {} --outdir {}".format(nspec, self.testdir)
+        quickbrick.main(quickbrick.parse(cmd.split()[1:]))
+        
+        f1a = desispec.io.read_frame('{}/brick-b-test1a.fits'.format(self.testdir))
+        f1b = desispec.io.read_frame('{}/brick-b-test1b.fits'.format(self.testdir))
+        f2 = desispec.io.read_frame('{}/brick-b-test2.fits'.format(self.testdir))
+        self.assertTrue(np.all(f1a.flux == f1b.flux))  #- same seed
+        self.assertTrue(np.all(f1a.ivar == f1b.ivar))
+        self.assertTrue(np.any(f1a.flux != f2.flux))   #- different seed
+
+        t1a = fits.getdata('{}/brick-b-test1a.fits'.format(self.testdir), '_TRUTH')
+        t1b = fits.getdata('{}/brick-b-test1b.fits'.format(self.testdir), '_TRUTH')
+        t2 = fits.getdata('{}/brick-b-test2.fits'.format(self.testdir), '_TRUTH')
+        self.assertTrue(np.all(t1a['TRUE_OBJTYPE'] == t1b['TRUE_OBJTYPE']))
+        self.assertTrue(np.all(t1b['TRUEZ'] == t1b['TRUEZ']))
+        self.assertTrue(np.any(t1a['TRUEZ'] != t2['TRUEZ']))
+
+    #- Test that brighter moon makes noisier spectra
+    @unittest.skipIf('TRAVIS_JOB_ID' in os.environ, 'Skipping memory hungry quickbrick/specsim test on Travis')
+    def test_quickbrick_moon(self):
+        nspec = 2
+        cmd = "quickbrick --seed 1 -b brightmoon --objtype BRIGHT_MIX -n {} --outdir {} --moon-phase 0.1".format(nspec, self.testdir)
```

Comment:
I would explicitly put the moon above the horizon here, instead of relying on a default.